docs_src/tutorial_1_hello_world.py

```diff
@@ -0,0 +1,20 @@
+from corva import Api, Corva, Event, State
+
+app = Corva()  # 1 initialize the app
+
+
+@app.stream  # 2 add decorator with needed event type to your function
+def user_job(event: Event, api: Api, state: State):
```

Comment:
Something to keep in mind is that people will probably be copying this code, or using one of our templates that will look like this, so a more useful name that `user_job` would be good.

Reply:
renamed to `stream_app`